code/drv_db/src/wattrex_driver_db/drv_db_dao_master.py

```diff
@@ -167,17 +180,34 @@ class DrvDbLinkConfigurationC(Base):
 
     CompDevID = Column(ForeignKey(DrvDbCompatibleDeviceC.CompDevID), primary_key=True,
                     nullable=False)
-    Property = Column(String(30), nullable=False)
+    Property = Column(String(30), primary_key = True, nullable=False)
     Value = Column(String(30), nullable=False)
 
-class DrvDbMeasuresDeclarationC(Base):
+class DrvDbAvailableMeasuresC(Base):
     '''
-    Class method to create a DRVDB model of database MeasuresDeclaration table.
+    Class method to create a DRVDB model of database AvailableMeasures table.
     '''
-    __tablename__ = 'MeasuresDeclaration'
+    __tablename__ = 'AvailableMeasures'
+    __table_args__ = (ForeignKeyConstraint(['CompDevID'], [DrvDbCompatibleDeviceC.CompDevID]),)
 
     MeasType = Column(MEDIUMINT(unsigned=True), primary_key=True)
-    MeasName = Column(String(20), nullable=False, unique=True)
+    CompDevID = Column(ForeignKey(DrvDbCompatibleDeviceC.CompDevID), nullable=False)
+    MeasName = Column(String(20), nullable=False)
+
+class DrvDbUsedMeasuresC(Base):
+    '''
+    Class method to create a DRVDB model of database UsedMeasures table.
+    '''
+    __tablename__ = 'UsedMeasures'
+    __table_args__ = (ForeignKeyConstraint(['MeasType'], [DrvDbAvailableMeasuresC.MeasType]),
+                      ForeignKeyConstraint(['DevID'], [DrvDbDetectedDeviceC.DevID]),
+                      ForeignKeyConstraint(['CSID'], [DrvDbCyclerStationC.CSID]),)
+
+    UsedMeasID = Column(MEDIUMINT(unsigned=True), primary_key=True, nullable=False)
+    CSID = Column(ForeignKey(DrvDbCyclerStationC.CSID), nullable=False)
```

Comment:
CSID, DevID and MeasType are unsigned

Reply:
As they are foreign keys there is no need to specify they are unsigned. It´s already defined in the respective class